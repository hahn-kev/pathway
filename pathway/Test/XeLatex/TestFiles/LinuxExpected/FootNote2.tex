\documentclass[a4paper]{article} 

\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage[margin=2cm,includeheadfoot]{geometry}
\begin{document} 
\pagestyle{plain} 
\font\scriptureTextscrBody="Times New Roman" at 12pt
\font\sectionscriptureTextscrBody="Times New Roman" at 12pt
\font\psectionscriptureTextscrBody="Times New Roman" at 12pt
\font\footnotepsectionscriptureTextscrBody="Times New Roman" at 9pt

\pagestyle{fancy} 
