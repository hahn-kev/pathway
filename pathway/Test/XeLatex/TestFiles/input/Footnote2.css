.Verse_Number
{
font-size: 12pt;
vertical-align: super;
}

.Paragraph
{
font-size: 12pt;
text-indent: .125pt;
}

.Alternate_Reading
{
font-size: 12pt;
font-style: italic;
}

.Endnote_General_Paragraph,
.Note_General_Paragraph {
    text-indent: 0pt;
    margin-left: 0pt;
    display: inline;
    display: footnote;
    display: prince-footnote;
    position: footnote;
    list-style-position: inside;
}
.Endnote_General_Paragraph::footnote-call,
.Note_General_Paragraph::footnote-call {
    color: purple;
    content: attr(title);
    vertical-align: super;
    vertical-align: top;
    line-height: none;
}
/*.Note_General_Paragraph::footnote-marker {
    font-weight: bold;
    content: string(chapter) string(verse) ' ';
    /* content: string(footnoteMarker); font-size: 6pt; vertical-align: super; color: purple;
    text-align: left;
} */
.Note_Target_Reference::footnote-marker {
    font-weight: bold;
}
.Endnote_General_Paragraph::footnote-call,
.Note_General_Paragraph::footnote-call {
    color: purple;
    content: attr(title);
    vertical-align: super;
    vertical-align: top;
    line-height: none;
}
.scrFootnoteMarker {
}
.Paragraph {
    text-indent: 12pt;
    margin-left: 0pt;
}

.Paragraph1 {
    margin-left: 0pt;
}


span {
    prince-text-replace: "<<" "\201C" ">>" "\201D" "<" "\2018" ">" "\2019";
}


.Verse_Number {
    vertical-align: super;
    text-indent: 0pt;
    margin-left: 0pt;
    string-set: verse content();
}

.Verse_Number1 {
    display: none;
}

span.reference :before {
	content: "(";
}
span.reference :after {
	content: ")";
}
