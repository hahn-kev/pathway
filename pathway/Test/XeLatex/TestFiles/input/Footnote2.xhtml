<html xml:lang="utf-8" lang="utf-8" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title />
    <link rel="stylesheet" href="Footnote2.css" type="text/css" />
    <meta name="description" content="Footnote2 exported by Greg Trihus on 2011-08-01T00:00:00-05:00" />
    <meta name="filename" content="Footnote2.xhtml" />
    <meta name="stylesheet" content="usfm.sty" />
    <meta name="fontName" content="Charis SIL" />
    <meta name="fontSize" content="12" />
  </head>
  <body class="scrBody">
    <div class="scrBook">
          <div class="Paragraph">
            <span class="Verse_Number" lang="zxx">2</span>
            <span lang="zxx">Abraham lɔ́kwɩɩ́ Isak. Isak lɔ́kwɩɩ́ Yakob. Yakob lɔ́kwɩɩ́ Yuda mʋa mʋ apio yinhɛ́</span>
            <span class="scrFootnoteMarker">
              <a href="#Footnote-MAT-1" />
            </span>
            <span class="Note_General_Paragraph" id="Footnote-MAT-1" title="a">
              <span class="Note_Target_Reference" lang="zxx">1:2 </span>
              <span class="Alternate_Reading" lang="zxx">Apio yinhɛ́ </span>
              <span lang="zxx">ánfɩ gyí Israel abusuan dúanyɔ ámʋ anáin nɩ.</span>
            </span>
            <span lang="zxx">. </span>
            <span class="Verse_Number" lang="zxx">3</span>
            <span lang="zxx">Yuda lɔ́kwɩɩ́ Peres mʋa Sera. Amʋ́ yín gyí Tamar. Peres lɔ́kwɩɩ́ Hesron. Hesron lɔ́kwɩɩ́ Ram. </span>
            <span class="Verse_Number" lang="zxx">4</span>
            <span lang="zxx">Ram lɔ́kwɩɩ́ Aminadab. Aminadab lɔ́kwɩɩ́ Nahson. Nahson lɔ́kwɩɩ́ Salmon. </span>
            <span class="Verse_Number" lang="zxx">5</span>
            <span lang="zxx">Salmon lɔ́kwɩɩ́ Boas. Boas mʋ yin gyí Rahab. Boas lɔ́kwɩɩ́ Obed. Obed mʋ yin gyí Rut. Obed lɔ́kwɩɩ́ Yisai. </span>
            <span class="Verse_Number" lang="zxx">6</span>
            <span lang="zxx">Yisai lɔ́kwɩɩ́ Owíe Dawid. </span>
          </div>
    </div>
  </body>
</html>