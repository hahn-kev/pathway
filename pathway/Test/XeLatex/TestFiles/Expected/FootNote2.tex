\documentclass{article} 
\usepackage{float}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}

\begin{document} 
\thispagestyle{empty} 
\font\AlternateReadingNoteGeneralParagraphParagraphscrBookscrBody="Charis SIL Compact/I" at 12pt
\font\spanNoteGeneralParagraphParagraphscrBookscrBody="Times New Roman" at 12pt
\font\NoteGeneralParagraphParagraphscrBookscrBody="Times New Roman" at 12pt
\font\spanParagraphscrBookscrBody="Times New Roman" at 12pt
\font\VerseNumberParagraphscrBookscrBody="Times New Roman" at 12pt
\font\ParagraphscrBookscrBody="Times New Roman" at 12pt
\font\scrBookscrBody="Times New Roman" at 12pt
\leftskip 0pt{\leftskip 0pt{
\noindent{\VerseNumberParagraphscrBookscrBody{$^{2 }$}}}\spanParagraphscrBookscrBody{Abraham lɔ́kwɩɩ́ Isak. Isak lɔ́kwɩɩ́ Yakob. Yakob lɔ́kwɩɩ́ Yuda mʋa mʋ apio yinhɛ́ }\leftskip 0pt{}  \footnote {1:2  Apio yinhɛ́  ánfɩ gyí Israel abusuan dúanyɔ ámʋ anáin nɩ. } \spanParagraphscrBookscrBody{. }\leftskip 0pt{\noindent{\VerseNumberParagraphscrBookscrBody{$^{3 }$}}}\spanParagraphscrBookscrBody{Yuda lɔ́kwɩɩ́ Peres mʋa Sera. Amʋ́ yín gyí Tamar. Peres lɔ́kwɩɩ́ Hesron. Hesron lɔ́kwɩɩ́ Ram. }\leftskip 0pt{\noindent{\VerseNumberParagraphscrBookscrBody{$^{4 }$}}}\spanParagraphscrBookscrBody{Ram lɔ́kwɩɩ́ Aminadab. Aminadab lɔ́kwɩɩ́ Nahson. Nahson lɔ́kwɩɩ́ Salmon. }\leftskip 0pt{\noindent{\VerseNumberParagraphscrBookscrBody{$^{5 }$}}}\spanParagraphscrBookscrBody{Salmon lɔ́kwɩɩ́ Boas. Boas mʋ yin gyí Rahab. Boas lɔ́kwɩɩ́ Obed. Obed mʋ yin gyí Rut. Obed lɔ́kwɩɩ́ Yisai. }\leftskip 0pt{\noindent{\VerseNumberParagraphscrBookscrBody{$^{6 }$}}}\spanParagraphscrBookscrBody{Yisai lɔ́kwɩɩ́ Owíe Dawid. }

\end{document}
